server/public/styles.css

```diff
@@ -0,0 +1,53 @@
+* {
+  font-family: Roboto, sans-serif;
+}
+
+.form-title {
+  margin-top: 43px;
+  font-size: 2.3rem;
+}
+
+.form-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+form {
+  width: 340px;
+}
```

Comment:
What about on Desktop? Your figma for desktop uses more space than that. Add a container class instead, like in your css responsive layout exercise.